backend/src/handlers/contact.ts

```diff
@@ -0,0 +1,29 @@
+import { Request, Response } from "express";
+import Contact from "../models/contact.model";
+import { handleIntegration } from "../API Manager/mondayAPI";
+import { searchImg } from "../API Manager/googleAPI";
+
+export const handleaddContact = async (
+  req: Request,
+  res: Response,
+) => {
+  try {
+    const img: string = await searchImg(req.body.searchFor);
+
+    const contact = await Contact.create({
+      fullname: req.body.values.fullname,
+      email: req.body.values.email,
+      title: req.body.values.title,
+      text: req.body.values.text,
+      img: img,
```

Comment:
Instead of passing the whole request object to the handler, just pass it with the needed arguments (fullname, email, etc.)